README.md

```diff
@@ -21,6 +21,11 @@ allprojects {
 
 ```gradle
 dependencies {
-    implementation 'com.github.Stremio:stremio-core-kotlin:1.0.21'
+    implementation 'com.github.Stremio:stremio-core-kotlin:1.0.22'
 }
 ```
+
+## Manual
```

Comment:
This should not have been added back, as with this new version we won't be generating a single aar, so it will be possible to consume it only via jitpack

Reply:
Oh right, good catch. Solving merge conflicts has always been difficult on this branch in particular :sweat_smile: